src/llmcompressor/entrypoints/oneshot.py

```diff
@@ -123,6 +126,13 @@ def __init__(
         # initialize the model and processor
         pre_process(model_args)
 
+        # offload to cpu if possible
+        if "cuda" in str(model_args.oneshot_device) and torch.cuda.is_available():
+            remove_hook_from_module(model_args.model, recurse=True)
+            offloaded_dispatch(model_args.model, model_args.oneshot_device)
```

Comment:
We offload the model before setting up the qparams, correct?
And qparams are initialized on the execution device https://github.com/neuralmagic/compressed-tensors/blob/3fb2844be33459e80ab813d90f0b7f98c3a9ce9a/src/compressed_tensors/quantization/lifecycle/initialize.py#L154? 

How do we make sure they're also offloaded?

Reply:
Newly registered parameters are offloaded if applicable
https://github.com/neuralmagic/compressed-tensors/blob/3fb2844be33459e80ab813d90f0b7f98c3a9ce9a/src/compressed_tensors/utils/offload.py#L212-L213